### PR TITLE
Migrate to Bzlmod

### PR DESCRIPTION
**Pull request by jadenPete**

### CODE REVIEW

File: MODULE.bazel
Comment (jadenPete) [inline MODULE.bazel]: - [ ] Wait until we've merged https://github.com/lucidsoftware/rules_scala/pull/74 and update this.

File: MODULE.bazel
Comment (jjudd) [inline MODULE.bazel]: I think we can mark this as `dev_dependency`

File: MODULE.bazel
Comment (jjudd) [inline MODULE.bazel]: I think we can mark this as `dev_dependency`

File: MODULE.bazel
Comment (jjudd) [inline MODULE.bazel]: Can this be moved up with the other `bazel_deps`?

Also I think we can mark this as `dev_dependency`

File: MODULE.bazel
Comment (jjudd) [inline MODULE.bazel]: Scope creep, but can we please update this to 1.10.4? Annex is already on that version, so it should be as simple as changing the version and updating the lockfile.

**Review by jjudd — COMMENTED** [review]
Mostly LGTM. Left a few comments.

File: MODULE.bazel
Comment (jadenPete) [reply MODULE.bazel]: See my comment here:
https://github.com/lucidsoftware/rules_scala/pull/74#discussion_r1873510258

File: MODULE.bazel
Comment (jadenPete) [reply MODULE.bazel]: See my comment here:
https://github.com/lucidsoftware/rules_scala/pull/74#discussion_r1873510258

File: MODULE.bazel
Comment (jadenPete) [reply MODULE.bazel]: See my comment here:
https://github.com/lucidsoftware/rules_play_routes/pull/57#discussion_r1873547143

File: MODULE.bazel
Comment (jadenPete) [reply MODULE.bazel]: Sure thing.

**Review by jjudd — APPROVED** [review]

**Comment by jadenPete** [comment]
Squashed.

**Pull request merged**
